docker-compose.ci.yml

```
@@ -0,0 +1,9 @@
+services:
+  prod:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    entrypoint: [ "sh", "/app/scripts/build-out.sh" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)